test/test_plot_results.jl

```diff
@@ -0,0 +1,68 @@
+IS.configure_logging(console_level = Logging.Info)
```

Comment:
This test uses a simulation to make the plots. Is there a way to test using only Operations? I say this because it would be the third test that relies on fully running simulations and increases a lot the testing times.

Reply:
Can the test load the outputs from a saved simulation?